ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/WiX/tasks/main.yml

```diff
@@ -35,25 +35,19 @@
 
 - name: Test if WiX is already installed
   win_stat:
-    path: 'C:\\Users\{{ Jenkins_Username }}\.dotnet/tools/wix'
+    path: 'C:\\Program Files\\dotnet\\wix.exe'
   register: wix_installed
   tags: Wix
 
 - name: Test WiX version
   # generates an output like 5.0.0+41e11442
   win_shell: wix --version
   register: installed_wix_version
-  become_user: "{{ Jenkins_Username }}"
-  become_method: ansible.builtin.runas
-  become: true
   when: wix_installed.stat.exists
   tags: Wix
 
 - name: Install Wix Toolset when not installed or version is not correct
-  win_shell: dotnet tool install --global wix --version {{ wix_version }}
-  become_user: "{{ Jenkins_Username }}"
-  become_method: ansible.builtin.runas
-  become: true
+  win_shell: dotnet tool install wix --version {{ wix_version }} --tool-path 'C:\\Program Files\\dotnet'
```

Comment:
Is the removal of `--global` here correct? To a non-expert that looks inconsistent with the goal of this PR

Reply:
Yes, strangely to install into a specific path, the --global-path has to be omitted, the --tool-path and --global parameters are mutually exclusive, by forcing a path that is available system wide, rather than under ../USERNAME/.dotnet/bin, the global flag is redundant.

Reply:
That's weird, but ok ;-)